### ISSUE TITLE
node 8+ not support

### ISSUE THREAD

**Issue by jjhesk** [tomaszczechowski/gulp-string-replace#11]
(node:6124) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.

gulp[6125]: ../src/node_contextify.cc:631:static void node::contextify::ContextifyScript::New(const FunctionCallbackInfo<v8::Value> &): Assertion `args[1]->IsString()' failed.
 1: node::Abort() [/usr/local/bin/node]
 2: node::Assert(char const* const (*) [4]) [/usr/local/bin/node]
 3: node::contextify::ContextifyScript::New(v8::FunctionCallbackInfo<v8::Value> const&) [/usr/local/bin/node]
 4: v8::internal::FunctionCallbackArguments::Call(v8::internal::CallHandlerInfo*) [/usr/local/bin/node]
 5: v8::internal::MaybeHandle<v8::internal::Object> v8::internal::(anonymous namespace)::HandleApiCallHelper<true>(v8::internal::Isolate*, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::HeapObject>, v8::internal::Handle<v8::internal::FunctionTemplateInfo>, v8::internal::Handle<v8::internal::Object>, v8::internal::BuiltinArguments) [/usr/local/bin/node]
 6: v8::internal::Builtin_Impl_HandleApiCall(v8::internal::BuiltinArguments, v8::internal::Isolate*) [/usr/local/bin/node]
 7: 0x38a69948427d
error Command failed with signal "SIGABRT".

https://github.com/tomaszczechowski/gulp-string-replace/blob/master/index.js#L77
https://nodejs.org/docs/latest-v9.x/api/buffer.html#buffer_class_method_buffer_allocunsafe_size
does it support higher version?

**Comment by tomaszczechowski** [comment]
Hi, 

It will be fixed soon. Let you know once new version would be released.

**Comment by tomaszczechowski** [comment]
I just checked on node v8.11.2, 7.0.0, 6.0.0, 4.9.1 and have not noticed this issue:

```bash
[23:10:33] Starting 'str-replace'...
[23:10:33] Finished 'str-replace' after 9.32 ms
[23:10:33] Starting 'default'...
[23:10:33] Finished 'default' after 36 μs
[23:10:33] Replaced: "world ||" in a file: app.js
```

even though I replaced `buffer` with `buffer.from` according to node recommendation. 

- Pls, download just released version 1.1.0 and check if the error remains.

- Pls rate project on GitHub 👍
